### ISSUE TITLE
add "stop backup" to menu

### ISSUE THREAD

**Issue by HinzundKunz** [spersson/Kup#28]
It would be nice if one could cancel/stop a running backup/sync process via the system tray menus.

**Comment by spersson** [comment]
Yes, I agree. I have just postponed it because I wanted to do some research first on potential problems with corrupt files as a result of brutally killing the rsync or bup process. Perhaps there even is a signal to send that would more gently ask the process to exit asap... Yep, just need some research... Anybody who knows more about this is welcome to comment!

**Comment by spersson** [comment]
I have brought it up and suggested changes here:
https://groups.google.com/forum/#!topic/bup-list/dIUw6BSb5Ks
No conclusion yet.
